boards/nrf52840-mdk/Makefile.include

```diff
@@ -11,4 +11,8 @@ else ifeq (openocd,$(PROGRAMMER))
   DEBUG_ADAPTER = dap
 endif
 
+# This can also be flashed with pyocd but not with jlink
+PROGRAMMERS_SUPPORTED_EXTRA ?= pyocd
+PROGRAMMERS_BLACKLIST ?= jlink
```

Comment:
Instead of this logic I would prefer just declaring `PROGRAMMERS_SUPPORTED` for this BOARD

Reply:
Agreed!